homeassistant/components/versasense/__init__.py

```diff
@@ -50,8 +50,8 @@ async def _configure_entities(hass, config, consumer):
     devices = await consumer.fetchDevices()
     _LOGGER.debug(devices)
 
-    sensor_info_list = []
-    switch_info_list = []
+    sensor_info_list = {}
```

Comment:
Optional cleanup.
```suggestion
    sensor_info = {}
```

Reply:
I was planning to do this in a follow-up PR - keep the focus on `list > dict` migration in this PR